components/event-service/internal/externalapi/externalapi.go

```diff
@@ -13,12 +14,14 @@ type SubscribedEventsHandler interface {
 	GetSubscribedEvents(w http.ResponseWriter, r *http.Request)
 }
 
-// NewHandler creates http.Handler(s) for the /v1/events /v1/subscribedevents and /v1/health endpoints
+// NewHandler creates http.Handler(s) for the /v1/events /v2/events /v1/subscribedevents and /v1/health endpoints
 func NewHandler(maxRequestSize int64, eventsClient subscribed.EventsClient) http.Handler {
 	router := mux.NewRouter()
 
 	router.Path("/{application}/v1/events").Handler(NewEventsHandler(maxRequestSize)).Methods(http.MethodPost)
```

Comment:
For consistency, please move the old API related files under a new directory `v1`.

Reply:
It's not very easy to do it, I'd create a follow-up ticket if I couldn't do it by the release